src/routes/governance/proposals-list.tsx

```diff
@@ -29,6 +29,7 @@ export const ProposalsList = ({ proposals }: ProposalsListProps) => {
     if (row.terms.change.__typename !== "UpdateNetworkParameter") return null;
 
     const type = row.terms.change.__typename;
+    const now = new Date();
```

Comment:
This will cause a re-render and re-computation every time the function is called. Can we reduce this by using [this pattern](https://github.com/vegaprotocol/token-frontend/blob/d08bedcf6b1633ec08afa6eb30c87b200ca9581a/src/components/epoch-countdown/epoch-countdown.tsx#L50-L61).

Reply:
thanks Dex, actually implemented a simpler fix suggested by Matt.